src/cmr/concepts/draft.js

```diff
@@ -125,16 +175,51 @@ export default class Draft extends Concept {
       'CMR-Request-Id'
     ])
 
-    // Add MetadataSpecification to the data
-    // eslint-disable-next-line no-param-reassign
-    data.metadataSpecification = this.metadataSpecification
-
     const metadata = {
-      metadataSpecification: this.metadataSpecification,
+      MetadataSpecification: this.metadataSpecification,
       nativeId: data.nativeId,
       ...data.metadata
     }
 
-    super.ingest(metadata, requestedKeys, permittedHeaders)
+    super.ingest(
+      metadata,
+      requestedKeys,
+      permittedHeaders
+    )
+  }
+
+  publish(data, requestedKeys, providedHeaders) {
+    const { ummVersion } = data
+
+    // eslint-disable-next-line no-param-reassign
```

Comment:
I see this 6 times in the repo -- wouldn't it be better to create a new `const` and remove the `eslint-disable`?